src/generator.ts

```diff
@@ -33,21 +33,36 @@ export type PromptOptions = {
 
 const assetsDir = path.join(__dirname, '..', 'assets');
 
-type CliArgumentKey = 'browser'
-  | 'no-browsers'
-  | 'no-examples'
-  | 'next'
-  | 'beta'
-  | 'ct'
-  | 'quiet'
-  | 'gha'
-  | 'install-deps'
-  | 'lang';
+export const allOptions = [
+  ['--help', 'print this message'],
+  ['--browser=<name>', "browsers to use in default config (default: 'chromium,firefox,webkit')"],
+  ['--browsers', 'download and use default browsers'],
+  ['--no-browsers', "do not download browsers (can be done manually via 'npx playwright install')"],
+  ['--no-examples', 'do not create example test files'],
+  ['--install-deps', 'install dependencies (default: false)'],
+  ['--next', 'install @next version of Playwright'],
+  ['--beta', 'install @beta version of Playwright'],
+  ['--ct', 'install Playwright Component testing'],
+  ['--quiet', 'do not ask for interactive input prompts'],
+  ['--gha', 'install GitHub Actions'],
+  ['--lang=<js>', "language to use (default: 'TypeScript'. Potential values: 'js', 'TypeScript')"],
+] as const;
+
+export type Option = (typeof allOptions)[number][0];
+
+export type OptionName = Exclude<
+  Option extends `--${infer BooleanOptionName}`
+    ? BooleanOptionName extends `${infer NonBooleanOptionName}=<${string}>`
+      ? NonBooleanOptionName
+      : BooleanOptionName
+    : never,
+  'help'
+>;
```

Comment:
use conditional types and `infer` types to avoid duplicating options in 2 places